app/views/incomes/editIncome.scala.html

```diff
@@ -14,14 +14,23 @@
  * limitations under the License.
  *@
 
-@(income: Form[uk.gov.hmrc.tai.forms.EditIncomeForm], hasMultipleIncomes: Boolean, employmentId: Int, payToDate: String = "0", payDate: Option[org.joda.time.LocalDate] = None,
-webChat: Boolean = false)(implicit request: Request[_], messages: Messages, user: controllers.auth.TaiUser,
-templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, partialRetriever: uk.gov.hmrc.play.partials.FormPartialRetriever)
+@(income: Form[uk.gov.hmrc.tai.forms.EditIncomeForm],
+    hasMultipleIncomes: Boolean,
+    employmentId: Int,
+    payToDate: String = "0",
+    payDate: Option[org.joda.time.LocalDate] = None,
+    webChat: Boolean = false)(implicit request: Request[_],
+    messages: Messages,
+    user: controllers.auth.TaiUser,
+    templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer,
+    partialRetriever: uk.gov.hmrc.play.partials.FormPartialRetriever
+)
 
 @import includes.ptaHeader
 @import uk.gov.hmrc.play.views.html.helpers._
 @import uk.gov.hmrc.play.language.LanguageUtils.Dates
 @import uk.gov.hmrc.time.TaxYearResolver
+@import includes._
```

Comment:
Do you need to include everything? 
Im not sure what the result of importing things you don't need is but for example when you make change to includes.yourTaxGraph (which this file doesn't use) you will probably need to recompile this file as it imports it.

You are also already importing `includes.ptaHeader`